pub_sub/javascript/checkout/package.json

```
@@ -0,0 +1,19 @@
+{
+    "name": "checkout",
+    "version": "1.0.0",
+    "description": "",
+    "main": "index.js",
+    "type": "module",
+    "scripts": {
+        "start": "node index.js",
+        "start:dapr": "dapr run --app-id checkout --app-protocol http --dapr-grpc-port 3501 --components-path ./components -- npm run start",
```

@XavierGeerinck should this be http?
```suggestion
        "start:dapr": "dapr run --app-id checkout --app-protocol http --dapr-http-port 3501 --components-path ./components -- npm run start",
```
```suggestion
        "start:dapr": "dapr run --app-id checkout --app-protocol http --dapr-grpc-port 3501 --components-path ./components -- npm run start",
```